src/shadowbox/docker/run_action.sh

```diff
@@ -36,6 +37,7 @@ declare -a docker_bindings=(
   -e SB_PRIVATE_KEY_FILE=${SB_PRIVATE_KEY_FILE}
 )
 
-echo "Running image ${SB_IMAGE}"
+readonly IMAGE=${SB_IMAGE:-quay.io/outline/shadowbox:stable}
```

Comment:
`SB_IMAGE` is not set by default.. This should default to `outline/shadowbox`, otherwise you will be running the production image, rather than the one you just built.